packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/OverviewTabPanel.tsx

```diff
@@ -120,6 +150,26 @@ export const OverviewTabPanel = ({
         ]}
       />
 
+      {Boolean(
+        creditNoteQuery?.data || isCreditNoteLoading || creditNoteError
+      ) && (
+        <Box
+          sx={{
+            '& > * + *': {
+              mt: 2,
+            },
+          }}
+        >
+          <Typography variant="subtitle2" sx={{ mb: 2 }}>{t(
+            i18n
+          )`Linked documents`}</Typography>
```

Comment:
I suggest not displaying the linked documents section until we are certain that at least one linked document exists for the given invoice. This can be easily implemented by checking the contents of `creditNoteIds` right from the start.

<img width="789" alt="image" src="https://github.com/user-attachments/assets/1236aa1d-113b-4292-b192-d7a0b9a40082">